runtime/doc/job_control.txt

```diff
@@ -102,23 +102,23 @@ function. Here's a more object-oriented version of the above:
 >
     let Shell = {}
 
-    function Shell.on_stdout(job_id, data)
+    function Shell.on_stdout(job_id, data) dict
```

Comment:
`dict` isn't need here or on any of the others below, since the function is being declared on a dictionary (`Shell`).